pkg/sql/sem/builtins/builtins.go

```diff
@@ -2484,7 +2484,9 @@ may increase either contention or retry errors, or both.`,
 				durationDelta := time.Duration(-beforeOffsetSecs) * time.Second
 				return tree.NewDTimeTZ(timetz.MakeTimeTZFromTime(tTime.In(loc).Add(durationDelta))), nil
 			},
-			Info:       "Treat given time without time zone as located in the specified time zone.",
+			Info: "Treat given time without time zone as located in the specified time zone.",
+			// TODO(mgartner): This overload might stable, not volatile.
+			// See: https://github.com/cockroachdb/cockroach/pull/48756#issuecomment-627672686
```

Comment:
this should fail https://github.com/cockroachdb/cockroach/blob/793ac98607ebec94b9ed3dbb15202e7de0c88ccc/pkg/sql/sem/builtins/all_builtins_test.go#L143 and you'll need to whitelist this difference from postgres.

Reply:
(setting `IgnoreVolatilityCheck: true` should do it)

Reply:
Good to know. Thanks for the heads-up!